Docs: Typos and edits in collections reviews (#79217)
ansible/ansible: docs/docsite/rst/community/collection_contributors/collection_reviewing.rst

When users report bugs, verify the behavior reported. Remember always to be kind with your feedback.

*  Did the user make a mistake in the code they put in the Steps to Reproduce issue section? We often see user errors reported as bugs.
*  Did the user assume an unexpected behavior? Ensure that the related documentation is clear. If not, the issue is useful to help us improve documentation.
*  Is there a minimal reproducer? If not, ask the reporter to reduce the complexity to help pinpoint the issue.

> *  Is the issue a consequence of a wrongly-configured environment?

Reviewer: ```suggestion
*  Is the issue a consequence of a misconfigured environment?
```

Reviewer: How about changing to "misconfigure" to avoid language that might blame the user?

*  If it seems to be a real bug, does the behaviour still exist in the most recent release or the development branch?
*  Reproduce the bug, or if you do not have a suitable infrastructure, ask other contributors to reproduce the bug.

Reviewing suggested changes
---------------------------

> When reviewing PRs, verify the suggested changes first. Verify that suggested changes do not:

Reviewer: ```suggestion
When reviewing PRs, verify that the suggested changes do not:
```

*  If a new plugin is added, it is one of the `allowed plugin types <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#modules-plugins>`_.
*  Documentation, examples, and return sections use FQCNs for the ``M(..)`` :ref:`format macros <module_documents_linking>` when referring to modules.
*  Modules and plugins from ansible-core use ``ansible.builtin.`` as an FQCN prefix when mentioned.

> *  When a new option, module, plugin, or return value is added, the corresponding documentation or return sections use ``version_added:`` containing the *collection* version in which they will be first released in.

Reviewer: ```suggestion
*  When a new option, module, plugin, or return value is added, the corresponding documentation or return sections use ``version_added:`` containing the *collection* version in which they will be first released.
```
